src/js/tabs/landing.js

```diff
@@ -38,7 +38,7 @@ landing.initialize = function (callback) {
             bottomSection.append(` <a href="#" lang="${element}" i18n="language_${element}"></a>`);
         });
 
-        bottomSection.find('a').each((index) => {
+        bottomSection.find('a').each(function(index) {
             let element = $(this);
             element.click(() => {
```

Comment:
```suggestion
            $(element).click(() => {
```

Also need to change this (if passing the param).

Reply:
Updated.